src/lib/net/encryption/Cargo.toml

```diff
@@ -5,3 +5,6 @@ edition = "2021"
 
 [dependencies]
 thiserror = { workspace = true }
+rsa = "0.9.6"
+der = "0.7.9"
+rand = "0.8.5"
```

Comment:
These dependencies should be placed in the workspace (project root) `Cargo.toml` and then imported as 
`crate = { workspace = true }`

See https://github.com/ferrumc-rs/ferrumc/blob/rewrite/v3/src/lib/README.MD